tests/test_bcrypt.py

```diff
@@ -146,6 +146,13 @@
         b"$2a$05$/OK.fbVrR/bpIqNJ5ianF.",
         b"$2a$05$/OK.fbVrR/bpIqNJ5ianF.Sa7shbm4.OzKpvFnX1pQLmQW96oUlCq"
     ),
+    (
+        b'}>\xb3\xfe\xf1\x8b\xa0\xe6(\xa2Lzq\xc3P\x7f\xcc\xc8b{\xf9\x14\xf6'
```

Comment:
I don't suppose there's a canonical source for test vectors containing null bytes in the password?

Reply:
I don't know any canonical source for test vectors, but I can easily generate other test cases.

Reply:
Is there another bcrypt implementation you can use to generate them to confirm that this code interoperates?

Reply:
Yes, the Perl implementation can be used for that.
I'll write a Perl script to create other test vectors and I will include the script and the generated vectors in my branch.

Reply:
Great, once we have that we should be in decent shape 👍